converter/commands/convert.go

```diff
@@ -54,6 +56,8 @@ func convert(env environment.Environment, noRotate bool) error {
 	//TODO: Decide on how to scale these options depending on the specs
 	//of the computer
 
+	//-------------------------------Input setup-------------------------------
```

Comment:
This function was getting huge so I put in some section headers. We ought to split this up into separate methods